### PR TITLE
(DOCS-3828) Adding event recommended tags

### PR DESCRIPTION
**Pull request by znbbz**
### What does this PR do?
Adding recommended tags for events

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/events/_index.md [hunk 1]
```diff
@@ -21,6 +24,7 @@ Datadog Events gives you a consolidated interface to search, analyze, and filter
 Without any additional setup, Datadog Events automatically gathers events that are collected by the Agent and installed integrations.
 
 More than 100 Datadog integrations support events collection, including [Kubernetes][1], [Docker][2], [Jenkins][3], [Chef][4], [Puppet][5], [AWS ECS][6] or [Autoscaling][7], [Sentry][8], and [Nagios][9]. 
+More than 100 Datadog integrations support events collection, including [Kubernetes][1], [Docker][2], [Jenkins][3], [Chef][4], [Puppet][5], [AWS ECS][6] or [Autoscaling][7], [Sentry][8], and [Nagios][9].
```
Comment (jtappa) [inline content/en/events/_index.md:27]: Looks like this line got duplicated accidentally

**Comment by jtappa** [comment]
Thanks @znbbz! I added an editorial review card for us to get it assigned during our next planning meeting.

File: content/en/events/guides/recommended_event_tags.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: Recommend Tagging Practices for Events
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:2]:
```suggestion
title: Best Practices For Tagging Events
```

File: content/en/events/guides/_index.md [hunk 3]
```diff
@@ -21,6 +21,10 @@ An event represents any record of activity noteworthy for engineers (devs, ops,
     {{< nextlink href="/api/v1/events/#post-an-event" >}}Datadog API{{< /nextlink >}}
 {{< /whatsnext >}}
 
+{{< whatsnext desc="Ensure your events are tagged:">}}
+{{< nextlink href="/events/guides/recommended_event_tags/" >}}Recommend Tagging Practices for Events{{< /nextlink >}}
```
Comment (alai97) [inline content/en/events/guides/_index.md:25]:
```suggestion
{{< nextlink href="/events/guides/recommended_event_tags/" >}}Best Practices for Tagging Events{{< /nextlink >}}
```

File: content/en/events/guides/recommended_event_tags.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+kind: documentation
+description: Recommended event tags and how to add them.
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:4]:
```suggestion
description: Learn about recommended event tags and how to add them.
```

File: content/en/events/guides/recommended_event_tags.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+aliases:
+    - /developers/taggingevents/
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:6]: This is a new file, unless this URL has been indexed, this alias does not work.
 
```suggestion
```

File: content/en/events/guides/recommended_event_tags.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+further_reading:
+- link: "/getting_started/tagging/assigning_tags"
+  tag: "Documentation"
+  text: "Assigning Tags"
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:10]:
```suggestion
  text: "Learn about assigning tags"
```

File: content/en/events/guides/recommended_event_tags.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+---
+
+It is recommended that all events should have the below tags to help you:
+- Identify potential issues faster
+- Locate other related events
+- Filter to more accurately e.g to a particular environment
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:17]:
```suggestion
## Overview

Datadog recommends tagging all events with attributes for the following benefits:

- Identify potential issues faster
- Locate related events
- Filter more accurately, for example, to a particular environment
```

File: content/en/events/guides/recommended_event_tags.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+If using the [API][1], they can be added within the tags param or to learn more about to how add them to certain integrations, check out [assigning_tags][2]
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:19]:
```suggestion
If you are using the [API][1], you can add tags within the `tags` parameter. For more information about adding tags with integrations, see [Assigning Tags][2].
```

File: content/en/events/guides/recommended_event_tags.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+For all alerts it recommended at minimum to use the above standard attributes and for the above integrations they will automatically have an aggregation_key added to them.
+
+| **Attribute**    | **Notes**                                                                                                                                                                  |
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| aggregation_key | All events coming from an alerts integration will have this attribute in order to deduplicate them. It can be found under the event attributes tab in the events explorer. |
+
+
+[1]: /api/latest/events/#post-an-event
+[2]: /getting_started/tagging/assigning_tags
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:55]:
```suggestion
[2]: /getting_started/tagging/assigning_tags/?tab=integrations
```

File: content/en/events/guides/recommended_event_tags.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+- [Webhooks][11]
+- [Zabbix][12]
+
+For all alerts it recommended at minimum to use the above standard attributes and for the above integrations they will automatically have an aggregation_key added to them.
+
+| **Attribute**    | **Notes**                                                                                                                                                                  |
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| aggregation_key | All events coming from an alerts integration will have this attribute in order to deduplicate them. It can be found under the event attributes tab in the events explorer. |
+
+
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:53]:
```suggestion
## Further Reading

{{< partial name="whats-next/whats-next.html" >}}

```

File: content/en/events/guides/recommended_event_tags.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+---
+
+It is recommended that all events should have the below tags to help you:
+- Identify potential issues faster
+- Locate other related events
+- Filter to more accurately e.g to a particular environment
+
+If using the [API][1], they can be added within the tags param or to learn more about to how add them to certain integrations, check out [assigning_tags][2]
+
+### Recommended Default Attributes (Type: all)
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:21]:
```suggestion
## Recommended attributes

These default attributes are available for all integration types.
```

File: content/en/events/guides/recommended_event_tags.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+
+| **Attribute** | **Notes**                                                                                                                                                                            |
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:23]:
```suggestion
| Attribute | Description                                                                                                                                                                            |
```

File: content/en/events/guides/recommended_event_tags.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+- [Sentry][7]
+- [SolarWinds Orion][8]
+- [Splunk][9]
+- [Sumo Logic][10]
+- [Webhooks][11]
+- [Zabbix][12]
+
+For all alerts it recommended at minimum to use the above standard attributes and for the above integrations they will automatically have an aggregation_key added to them.
+
+| **Attribute**    | **Notes**                                                                                                                                                                  |
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:49]:
```suggestion
| Attribute    | Description                                                                                                                                                                  |
```

File: content/en/events/guides/recommended_event_tags.md [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+| env          | The environment in which the event is from e.g production, edge, staging. <br>Enables you to ensure that events from a lower environment are not confused to be of a higher priority |
+| service      | The name of the service                                                                                                                                                              |
+| host         | The host name that either the service, pod, cluster, etc is running on                                                                                                               |
+| team         | The team which owns the event, so that they can be later notified if need be                                                                                                         |
+| version      | The build, service, etc version so that you know that an outage for instance is only related to a particular version released                                                        |
+
+
+### Type: Alert
+
+The current supported alerts integrations, are:
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:34]:
```suggestion
### Alert attributes

This attribute is available for the following alerts integrations:
```

File: content/en/events/guides/recommended_event_tags.md [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+- [New Relic][5]
+- [Prometheus AlertManager][6]
+- [Sentry][7]
+- [SolarWinds Orion][8]
+- [Splunk][9]
+- [Sumo Logic][10]
+- [Webhooks][11]
+- [Zabbix][12]
+
+For all alerts it recommended at minimum to use the above standard attributes and for the above integrations they will automatically have an aggregation_key added to them.
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:47]:
```suggestion

Datadog recommends using the recommended attributes at a minimum for all alerts. The alerts integrations automatically have an `aggregation_key`. 
```

File: content/en/events/guides/recommended_event_tags.md [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+[3]: /integrations/amazon_web_services
+[4]: /integrations/nagios
+[5]: /integrations/new_relic
+[6]: /integrations/prometheus
+[7]: /integrations/sentry
+[8]: /integrations/solarwinds
+[9]: /integrations/splunk
+[10]: /integrations/sumo_logic
+[11]: /integrations/webhooks
+[12]: /integrations/zabbix
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:65]:
```suggestion
[12]: /integrations/zabbix
[13]: /events/explorer/#attributes-tab
```

File: content/en/events/guides/recommended_event_tags.md [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+- [Splunk][9]
+- [Sumo Logic][10]
+- [Webhooks][11]
+- [Zabbix][12]
+
+For all alerts it recommended at minimum to use the above standard attributes and for the above integrations they will automatically have an aggregation_key added to them.
+
+| **Attribute**    | **Notes**                                                                                                                                                                  |
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| aggregation_key | All events coming from an alerts integration will have this attribute in order to deduplicate them. It can be found under the event attributes tab in the events explorer. |
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:51]:
```suggestion
| aggregation_key | All events coming from an alerts integration have this attribute to de-duplicate them. This appears on the **Event Attributes** tab when you click on an attribute in the [Events Explorer][13]. |
```

File: content/en/events/guides/recommended_event_tags.md [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+- Locate other related events
+- Filter to more accurately e.g to a particular environment
+
+If using the [API][1], they can be added within the tags param or to learn more about to how add them to certain integrations, check out [assigning_tags][2]
+
+### Recommended Default Attributes (Type: all)
+
+| **Attribute** | **Notes**                                                                                                                                                                            |
+|--------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| env          | The environment in which the event is from e.g production, edge, staging. <br>Enables you to ensure that events from a lower environment are not confused to be of a higher priority |
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:25]:
```suggestion
| env          | The environment in which the event is from, such as production, edge, or staging. <br>This enables you to ensure that events from a lower environment are not confused as high priority. |
```

File: content/en/events/guides/recommended_event_tags.md [hunk 19]
```diff
@@ -0,0 +1,65 @@
...
+| service      | The name of the service                                                                                                                                                              |
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:26]:
```suggestion
| service      | The service name.                                                                                                                                                              |
```

File: content/en/events/guides/recommended_event_tags.md [hunk 20]
```diff
@@ -0,0 +1,65 @@
...
+| host         | The host name that either the service, pod, cluster, etc is running on                                                                                                               |
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:27]:
```suggestion
| host         | The host name that the service, pod, or cluster is running on.                                                                                                               |
```

File: content/en/events/guides/recommended_event_tags.md [hunk 21]
```diff
@@ -0,0 +1,65 @@
...
+| team         | The team which owns the event, so that they can be later notified if need be                                                                                                         |
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:28]:
```suggestion
| team         | The team that owns the event so they can be notified if need be.                                                                                                         |
```

File: content/en/events/guides/recommended_event_tags.md [hunk 22]
```diff
@@ -0,0 +1,65 @@
...
+| version      | The build, service, etc version so that you know that an outage for instance is only related to a particular version released                                                        |
```
Comment (alai97) [inline content/en/events/guides/recommended_event_tags.md:29]:
```suggestion
| version      | The build or service version. This allows you to identify, for example, if an outage is related to a particular release version.                                                        |
```

File: content/en/events/_index.md (hunk 1 above)
Comment (alai97) [reply content/en/events/_index.md:27]:
```suggestion
```

File: content/en/events/_index.md [hunk 23]
```diff
@@ -8,6 +8,9 @@ further_reading:
 - link: "/api/v1/events/"
   tag: "Documentation"
   text: "Datadog Events API"
+- link: "events/guides/recommended_event_tags/"
+  tag: "Documentation"
+  text: "Recommend Tagging Practices for Events"
```
Comment (alai97) [inline content/en/events/_index.md:13]:
```suggestion
- link: "/events/guides/recommended_event_tags/"
  tag: "Documentation"
  text: "Best Practices for Tagging Events"
```

**Review by alai97 — CHANGES_REQUESTED** [review]

**Comment by znbbz** [comment]
Hi!  @alai97 & @jtappa 

I just wanted to follow up on this. let me know if it looks good to you

**Review by alai97 — APPROVED** [review]

**Review by jtappa — APPROVED** [review]

**Pull request merged**